src/storage/hummock_test/src/test_utils.rs

```diff
@@ -296,7 +299,7 @@ pub async fn with_hummock_storage_v1() -> (HummockStorageV1, Arc<MockHummockMeta
     .await
     .unwrap();
 
-    register_test_tables(
+    register_test_tables_with_id(
```

Comment:
There are two options (prefer 2), but I'm not sure if it makes sense, please correct me if I'm wrong.

1. Named the functions in `test_utils` with `test`,
2. Remove the function_name `test` in `test_utils` and use test_utils::func to call it externally?

Reply:
discussed offline: 1) is better since we cannot enforce using `test_utils::func` to call the func.